public/talos/v1.11/security/talos-security-checklist.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Talos Security Checklist
+description: A practical checklist for securing Talos Linux clusters.
+---
+
+Talos is designed to be secure by default, no SSH, no interactive shell, an API-driven OS, and an immutable filesystem.
+
+Still, there are several additional steps you can take to strengthen the security posture of your cluster.
+
+This checklist highlights the key areas to review and configure when running Talos in production environments.
+
+## Protect secrets and sensitive information
+
+Talos stores certificates, keys, and other sensitive configuration data inside the machine configuration. To keep this material secure, make sure you:
+
+- Encrypt secrets stored in your Talos machine config.
+
+- Limit who has access to machine configuration files, as they contain cluster CA keys and API credentials.
+
+- Rotate certificates regularly, including cluster CA, API server certificates, and kubelet client certificates.
+
+- Avoid embedding long-lived credentials into your config; prefer short-lived tokens or rotate them often.
+
+For more details, see:
+
+- [Certificate Authority Rotation](../security/ca-rotation)
+- [Generating a Secrets Bundle](../getting-started/prodnotes#step-5%3A-generate-secrets-bundle)
+- [Talos Certificate Management](../security/cert-management)
+- [`talosctl rotate ca` Reference](../reference/cli#talosctl-rotate-ca)
+
+## Enable disk encryption
+
+Encrypting disks helps ensure that sensitive data cannot be accessed if the underlying hardware is lost or compromised. In Talos, you can enable several layers of disk protection:
+
+- **TPM-backed disk encryption**: Uses the node’s TPM to automatically unlock encrypted volumes during boot.
+
+- **SecureBoot + TPM for verified boot paths**: Confirms that the kernel and operating system haven’t been tampered with before the system starts.
+
+- **Encrypted user volumes**: Protects data stored on node-local disks used by workloads.
+
+You can configure disk encryption manually or manage it centrally through Omni.
+
+For more details, see [Disk Encryption](../configure-your-talos-cluster/storage-and-disk-management/disk-encryption).
+
+## Keep Talos machines up to date
+
+Talos has a small and predictable upgrade surface, and each release typically includes important changes such as:
+
+- Kernel security patches
+
+- Container runtime updates
+
+- Enhancements to the Talos API
+
+- CVE and security fixes across the operating system
+
+To stay secure and up to date, make sure you upgrade regularly. You can upgrade individual nodes wit
```

Comment:
Maybe mention Talos' release cycle (3x per year) for minor versions. I'm not sure if there's a schedule for patch releases but we do have a SLA for releasing patches for CVE